### DISCUSSION TITLE
system test update - Dec 2023

### DISCUSSION THREAD

**Discussion by PWPInnovator898** [kriswilk/buzzah#10] [Show and tell]
Hi Kris;
After a fair bit of vibration parameter tweaking, I'm using the system 2 to 3 hrs per day w/ moderately encouraging results.
I remain cautiously optimistic.
Attached is a pic of the system contained in a runners pack, which makes it relatively mobile.  :-)
We are working on new tactor holders now (smaller) to enhance durability/comfort.
The next revision may include independent controllers for each hand and to reduce size, as Dr. Tass at Stanford seemed to have confirmed (in a webinar) that treatment does not have to be concurrent on both hands.
Thanks again for your excellent work and the generosity of your expertise on this platform!   
I hope it continues to benefit many people!
![IMG_3107](https://github.com/kriswilk/buzzah/assets/128418916/d606d332-cc92-41a5-90de-9bd4c8125d72)
Sincerely, Phillip

**Comment by kriswilk** [comment]
Hey Phillip, thanks for your update. Your hardware implementation looks really great -- very streamlined! I'm sure it will give others who are interested in building their own some ideas. 👍

**Reply by DBGuy123** [reply]
Hi do you or anone watching  haave a file on the design of the tactor housing in 
STEP or SLDPRT files. I have been in communication with Neurosmith24 but his file is in a different format than my printer is used to.

**Comment by PWPInnovator898** [comment]
Thanks Kris, and Happy New Year!

On Fri, Jan 19, 2024 at 8:03 AM Kris Wilk ***@***.***> wrote:

> Hey Phillip, thanks for your update. Your hardware implementation looks
> really great -- very streamlined! I'm sure it will give others who are
> interested in building their own some ideas. 👍
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kriswilk/buzzah/discussions/10#discussioncomment-8182602>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6TYIZB3W4J54XE2E5HWZLLYPKDM3AVCNFSM6AAAAABAWTVHVWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DCOBSGYYDE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Thanks and best regards,
Phillip Callahan
